ec2utils/ec2deprecateimg/lib/ec2utils/ec2deprecateimg.py

```diff
@@ -73,30 +73,14 @@ def __init__(
     # ---------------------------------------------------------------------
     def _find_images_by_id(self, image_id, filter_replacement_image=None):
```

Comment:
Why not just get rid of this function, and call `utils.find_images_by_id()` directly?